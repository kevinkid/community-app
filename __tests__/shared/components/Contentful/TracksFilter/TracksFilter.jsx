import React from 'react';
import Renderer from 'react-test-renderer/shallow';
import { TracksFilterInner } from 'components/Contentful/TracksFilter/TracksFilter';

test('Matches shallow shapshot', () => {
  const renderer = new Renderer();
  const MOCK_PROPS = {
    theme: {},
    startDate: '2019-08-01T07:21:20.249Z',
    endDate: '2019-09-01T07:21:20.249Z',
  };
  renderer.render(<TracksFilterInner {...MOCK_PROPS} />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
