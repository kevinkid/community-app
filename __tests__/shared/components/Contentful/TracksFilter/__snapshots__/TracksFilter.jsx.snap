// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow shapshot 1`] = `
<div>
  <div
    className="undefined undefined"
  >
    <span>
      filter
    </span>
    <button
      onClick={[Function]}
      type="button"
    >
      <IconCloseBig
        height="14"
        viewBox="0 0 14 14"
        width="14"
        xmlns="http://www.w3.org/2000/svg"
      />
    </button>
  </div>
  <ThemedTracksTagsInner
    composeAdhocTheme="deeply"
    composeContextTheme="softly"
    mapThemrProps={[Function]}
    onAddNewTag={[Function]}
    onRemoveTag={[Function]}
    tags={Array []}
    themePriority="adhoc-context-default"
  />
  <div>
    <ThemedTracksAuthorInner
      className=""
      composeAdhocTheme="deeply"
      composeContextTheme="softly"
      mapThemrProps={[Function]}
      onSelected={[Function]}
      options={
        Array [
          "All authors",
        ]
      }
      selected="All authors"
      themePriority="adhoc-context-default"
    />
    <ThemedTracksDateInner
      className=""
      composeAdhocTheme="deeply"
      composeContextTheme="softly"
      endDate="2019-09-01T07:21:20.249Z"
      mapThemrProps={[Function]}
      onSelectEndDate={[Function]}
      onSelectStartDate={[Function]}
      startDate="2019-08-01T07:21:20.249Z"
      themePriority="adhoc-context-default"
    />
  </div>
  <div>
    <button
      onClick={[Function]}
      type="button"
    >
      RESET
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      APPLY FILTER
    </button>
  </div>
</div>
`;
